src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,21 +20,47 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
-/**
- * Servlet that returns some example content.
- * comments data
- */
+import org.json.simple.JSONObject;
+/**Servlet responsible for storing Youtube Video and Displaying the details of the Youtube Video*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final String TIMESTAMP = "timestamp";
+  private static final String TITLE = "title";
+  private static final String SENTIMENT = "sentiment";
+  private static final String UPVOTES = "upvotes";
+  private static final String URL = "url";
+  private static List<String> threadTitles = new ArrayList<String>();
+  private static List<Double> threadSentiments = new ArrayList<Double>();
+  private static List<Integer> threadUpvotes = new ArrayList<Integer>();
+  private static List<String> threadUrls = new ArrayList<String>();
+  private static JSONObject threadInfoList = new JSONObject();
+  private final Gson gson = new Gson();
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    for (int i = 0; i < 5; i++) {
+      threadTitles.add(AnalyzedVideo.getRandomTitle());
```

Comment:
To confirm: do you want new mock posts to be added on each doGet? If not, convert these Lists to local variables.

Reply:
I would like new mock posts to be added on each doGet.